### PR TITLE
[10.0] add new addon to track all customer deliveries impacted by a lot

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**

### CODE REVIEW

File: stock_lot_sale_tracking/security/security.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.model.access" id="wip_lot_access_everybody">
```
Comment (rousseldenis) [inline stock_lot_sale_tracking/security/security.xml]: @Cedric-Pigeon Remove 'wip' ?

Comment (Cedric-Pigeon) [reply stock_lot_sale_tracking/security/security.xml]: @rousseldenis Done !

**Comment by Cedric-Pigeon** [comment]
@rousseldenis Done !

**Review by gurneyalex — APPROVED** [review]

File: stock_lot_sale_tracking/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+    'name': 'This addon allows to retrieve all customer deliveries impacted '
```
Comment (LoisRForgeFlow) [inline stock_lot_sale_tracking/__manifest__.py:6]: this should be the summary, right? and then you will also need to add the proper name.

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**